dist/angular-breadcrumb.js

```diff
@@ -212,6 +216,7 @@ function BreadcrumbDirective($interpolate, $breadcrumb, $rootScope) {
                     scope.steps = $breadcrumb.getStatesChain();
                     angular.forEach(scope.steps, function (step) {
                         if (step.ncyBreadcrumb && step.ncyBreadcrumb.label) {
+                            viewScope.ncyStateParams = $breadcrumb.getStateParams();
```

Comment:
I forget an important part:

Before doing anything in your branch history, please can you revert changes on `release/angular-breadcrumb.js` and `release/angular-breadcrumb.min.js` ? 

I change these file only when I made a new release (it allows users to always have a stable version when they checkout the master).